From: Alexis Murzeau <amubtdx@gmail.com>
Date: Thu, 31 Dec 2020 13:13:14 +0100
Subject: Remove sphinx 3.0 requirement

Sphinx 3.0 was required only for the furo theme, which is not used.
Streamlink docs build fine with older sphinx versions and sphinx 3.0
is not available in Buster.

Forwarded: not-needed
Author: Alexis Murzeau <amubtdx@gmail.com>
---
 docs/conf.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/conf.py b/docs/conf.py
index 979bd82..6fb3be1 100644
--- a/docs/conf.py
+++ b/docs/conf.py
@@ -13,7 +13,7 @@ sys.path.insert(0, os.path.abspath('.'))
 # -- General configuration -----------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
-needs_sphinx = '3.0'
+#needs_sphinx = '3.0'
 
 # Add any Sphinx extension module names here, as strings. They can be extensions
 # coming with Sphinx (named 'sphinx.ext.*') or your custom ones.
